scvi/external/stereoscope/_model.py

```diff
@@ -0,0 +1,138 @@
+from typing import Tuple
+
+import numpy as np
+from anndata import AnnData
+
+from scvi._compat import Literal
+from scvi.dataloaders import ScviDataLoader
+from scvi.external.stereoscope._module import RNADeconv, SpatialDeconv
+from scvi.lightning import VAETask
+from scvi.model.base import BaseModelClass
+
+
+class RNAStereoscope(BaseModelClass):
+    """
+    Reimplementation of Stereoscope for deconvolution of spatial transcriptomics from single-cell transcriptomics.
+
```

Comment:
Add a reference to the publication like in SCVI... need to add the entry into the references.rst file in docs